car.capnp

```diff
@@ -182,6 +182,7 @@ struct CarState {
       altButton3 @8;
     }
   }
+  blindspot @27 :Bool; # Is there something blocking the lane change
```

Comment:
I think at least distinguishing between left VS right should be required.
Also, maybe anything that has to do with perception should be handled by radard. I'll think about this.